tutorials/shield_multiphysics/doc/content/user_workshop/problem/step03.md

```diff
@@ -0,0 +1,91 @@
+# Step 3: Boundary conditions id=step03
+
+!---
+
+Fixed temperatures at the boundary conditions are simple but not realistic. We want to have:
+
+- a fixed heat flux from the micro-reactor inside the concrete cavity
+
+- natural convection boundary conditions with the air around the shield
+
+- convective boundary conditions with the water block
+
```

Comment:
Would be nice to add a figure illustrating the BCs

Reply:
done